kotlin-app-recipient/src/test/kotlin/recipient/controller/SenderInvoiceReflectPostTest.kt

```diff
@@ -128,10 +133,27 @@ class SenderInvoiceReflectPostTest {
         return handle.createQuery(sql).mapTo(InvoiceRow::class.java).list()
     }
 
+    private fun getInvoiceMemos(handle: Handle): List<InvoiceMemoRow> {
+        val sql =
+            """
+            SELECT
+                invoice_uuid,
+                memo
+            FROM invoice_memo
+            """.trimIndent()
+
+        return handle.createQuery(sql).mapTo(InvoiceMemoRow::class.java).list()
```

Comment:
一件前提であれば 
```suggestion
        return handle.createQuery(sql).mapTo(InvoiceMemoRow::class.java).singleOrNull()
```

っていうメソッドを使ったりします